acid/templates/base.html

```diff
@@ -42,6 +42,18 @@
                 <li class="nav-item {% if request.path.startswith('/builds') %}active{% endif %}">
                     <a class="nav-link" href="{{ url_for('builds.show_builds_history') }}">Builds</a>
                 </li>
+                <li>
+                    <div class="col ml-5"></div>
+                </li>
+                <li class="nav-item ml-3">
```

Comment:
Change styling of this item. When an admin is logged in it overlaps with other elements in navbar.